piksi_pps_sync/install/install.sh

```diff
@@ -1,43 +1,80 @@
 #!/bin/bash
-GPIO_PIN=250
+echo "Please enter the PPS GPIO pin number, e.g., 250?"
+read GPIO_PIN
+echo "PPS is triggered on pin ${GPIO_PIN}."
 
-DEVICE='/dev/ttyS5'
-BAUD=230400
+echo "Please enter the NMEA UART device, e.g., /dev/ttyS5?"
+read DEVICE
+echo "Using serial port ${DEVICE}."
+
+echo "Please enter the serial port baud rate, e.g., 115200?"
+read BAUD
+echo "Setting baud rate to ${BAUD}."
 
 # Install PPS
 cd ..
 cd pps-gpio-modprobe
+sudo apt-get install linux-headers-$(uname -r)
 make clean
 make
 sudo cp pps-gpio-modprobe.ko /lib/modules/$(uname -r)/kernel/drivers/pps/clients/
 sudo depmod
-sudo sh -c 'echo "pps-gpio-modprobe" >> /etc/modules-load.d/10-pps-gpio-modprobe.conf'
-sudo sh -c "echo 'options pps-gpio-modprobe gpio=${GPIO_PIN}' >> /etc/modprobe.d/10-pps-gpio-modprobe.conf"
 cd ..
 
 # Install NMEA GPS
 sudo apt install gpsd -y
 sudo cp cfg/gpsd /etc/default/gpsd
-sudo dpkg-reconfigure gpsd
+echo "Do you wish to configure gpsd? [y or Y to accept]"
+read configure_gpsd
+if [[ $configure_gpsd == "Y" || $configure_gpsd == "y" ]]; then
+  echo "Configuring /etc/default/gpsd"
+  sudo rm /etc/default/gpsd
+  sudo sh -c "echo '# Start the gpsd daemon automatically at boot time.' >> /etc/default/gpsd"
+  sudo sh -c "echo 'START_DAEMON=\"true\"' >> /etc/default/gpsd"
+  sudo sh -c "echo '# Use USB hotplugging to add new USB devices automatically to the daemon.' >> /etc/default/gpsd"
+  sudo sh -c "echo 'USBAUTO=\"false\"' >> /etc/default/gpsd"
+  sudo sh -c "echo '# Devices gpsd should collect to at boot time.' >> /etc/default/gpsd"
+  sudo sh -c "echo '# They need to be read/writeable, either by user gpsd or the group dialout.' >> /etc/default/gpsd"
+  sudo sh -c "echo 'DEVICES=\"${DEVICE}\"' >> /etc/default/gpsd"
+  sudo sh -c "echo '# Other options you want to pass to gpsd.' >> /etc/default/gpsd"
+  sudo sh -c "echo 'GPSD_OPTIONS=\"-n -r\"' >> /etc/default/gpsd"
+  sudo sh -c "echo '' >> /etc/default/gpsd"
+  sudo sh -c "echo 'GPSD_SOCKET=\"/var/run/gpsd.sock\"' >> /etc/default/gpsd"
+  sudo chmod +x /etc/rc.local
```

Comment:
could use tee for writing the config file (way more readable), made a quick example, see here
https://gist.github.com/michaelpantic/eaa081c78900f97d7a42ce15558f55ec